tests/rule.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Basic
```

Comment:
Can we alsom move `features/` dir somewhere to `tests/` dir? It's a little bit unclear why they're in root dir.